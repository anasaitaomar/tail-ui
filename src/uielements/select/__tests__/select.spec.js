import React from 'react';
import { render } from 'test-utils';
import  Select from 'uielements/select/select.component.js';

it('should take a snapshot', () => {
    const { asFragment } = render(<Select />)
    
    expect(asFragment(<Select />)).toMatchSnapshot()
});
