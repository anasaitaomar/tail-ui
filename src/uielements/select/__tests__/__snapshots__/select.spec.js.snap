// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <select
    class="pt-2"
    id="tailui-select"
  />
</DocumentFragment>
`;
